pandas/tests/series/test_repr.py

```diff
@@ -240,6 +240,15 @@ def test_series_repr_nat(self):
         )
         assert result == expected
 
+    def test_float_repr_in_frame(self):
```

Comment:
can you remove _in_frame (as this a series).

Reply:
Ok, sure.